apps/www/registry/default/ui/calendar.tsx

```diff
@@ -3,9 +3,9 @@
 import * as React from "react"
 import { ChevronLeft, ChevronRight } from "lucide-react"
```

Comment:
should be `ChevronLeftIcon` and `ChevronRightIcon` here?

Reply:
Its correct - 
<img width="688" alt="image" src="https://github.com/shadcn-ui/ui/assets/7449806/2d8382c3-19ca-471e-8c16-4a74d7e32e52">

https://lucide.dev/icons/chevron-left